src/components/CommentsContainer.tsx

```diff
@@ -0,0 +1,26 @@
+import CommentsList from './CommentsList';
+import Header from './Header';
+import { classes } from '../style/main.st.css';
+import PostReply from './PostReply';
+
+interface postInfo {
+    userName: string;
+    profilePicture: string;
+    postDescription: string;
+}
+
+const CommentsContainer = ({ userName, postDescription, profilePicture }: postInfo) => {
```

Comment:
Change this to something more generic, the header and post reply sections dont really relate to the comments in any way

Reply:
postLaout?